src/main/java/duke/ui/NotgptJavaFX.java

```diff
@@ -0,0 +1,132 @@
+package duke.ui;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.*;
+import javafx.scene.layout.*;
+import javafx.stage.Stage;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontWeight;
+
+public class NotgptJavaFX extends Application {
+
+    private Parser parser = new Parser();
+    private VBox chatBox;
+    private TextField inputField;
+    private Button sendButton;
+    private Image userAvatar;
+    private Image botAvatar;
```

Comment:
Naming here is pretty good. It is quite obvious what each variable represents without additional comments.